fluent/exceptions.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
the file is called `errors` in JS and Rust impl.

Reply:
`exceptions.py` seems to be more common than `errors.py` but it looks like both have plenty of examples (according to GitHub search) and for consistency with other implementations I'll change it.